lib/msf/ui/console/command_dispatcher/creds.rb

```diff
@@ -535,6 +542,11 @@ def cmd_creds_tabs(str, words)
 
   def filtered_query(query, opts, origin_ranges, host_ranges)
     query.each do |core|
+      # we will not show the cracked password in a seperate row instead we will show in seperate column
+      if core.origin.kind_of?(Metasploit::Credential::Origin::CrackedPassword)
+        next
+      end
+
```

Comment:
Not sure if this is related to this but was getting some inconsistencies whereby I'd delete a cred entry and then try to add the same one back in again, but I'd get errors that the credential already existed even though it should have been removed from the database 😕

Reply:
@gwillcox-r7 this may be a separate bug we are currently working on tracking down, I suspect that some credential objects are being orphaned due to changes in how Rails 7 handles them.

Reply:
that has been a bug for a LOOOOOOOOOOOOONG time. I'm pretty sure I documented it back when I originally rewrote the crackers a few years ago

Reply:
@jmartin-r7 Has there been a bug entry to track this bug? If so I'm happy to mark this as resolved for now, just want to make sure we are tracking this.

Reply:
The deleted cred add issue is being worked now, you can consider it separate from this PR.